client/src/components/ProductDetails/DropDownsAndButtons.jsx

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import './prodStyles.css';
+
+
+function DropDownsAndButtons ({productStyles, currentProductID, currentStyleID}) {
+
+  const [inventory, updateInventory] = useState(5);
+
+  const getCurrentProductObject = (targetId, allProducts) => {
```

Comment:
Would it be possible to avoid duplicating this logic (I think it's duplicate)? Perhaps you could take care of this processing of data in a parent component and then pass the processed data down to the children components

Reply:
As stated in my previous comment, I plan to move all state logic for all of my components over to the ProductOverview container. 

For now I opted to accrue tech dept in order to focus on making the components styling more presentable- mainly implementing flexbox